.github/workflows/ci.yml

```diff
@@ -0,0 +1,112 @@
+name: Pipeline
+
+on:
+  pull_request:
+    branches:
+      - master
+  push:
+    branches:
+      - test_task
+
+env:
+  REGISTRY: ghcr.io
+  IMAGE_NAME: ${{ github.repository }}
+
+jobs:
+  ci:
+    runs-on: ubuntu-latest
+    steps:
+      - name: start timer
+        run: |
+         echo "START_TIME=$(date +%s)" >> $GITHUB_ENV
+
+      - name: checkout
+        uses: actions/checkout@v4
+
+      - name: get node.js version from package.json
+        id: get-versions
+        run: echo "::set-output name=node::$(jq -r .version ${{ github.workspace }}/angular/package.json)"
```

Comment:
LEt's use GITHUB_OUTPUT env var here? https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/workflow-commands-for-github-actions#setting-an-output-parameter